Tests for run_becnhmark.py failed for --test_mode

After recent changes benchmark tests failed

```
=================================== FAILURES ===================================
________________________ TestRunBenchmark.test_get_sock ________________________

self = <test.test_run_benchmark.TestRunBenchmark object at 0x7f85591ca470>
return_run_benchmark_obj = <prepare_env_test_mode.run_benchmark.RunBenchmark object at 0x7f85591a5eb8>

    def test_get_sock(self, return_run_benchmark_obj):
>       assert "sock" in return_run_benchmark_obj.get_sock()
E       TypeError: get_sock() missing 1 required positional argument: 'basedir'

test_run_benchmark.py:10: TypeError
```

--- a/prepare_env_test_mode/run_benchmark.py
+++ b/prepare_env_test_mode/run_benchmark.py
@@ -57,7 +57,6 @@ class RunBenchmark:
         # Running sysbench here; The parameters hard coded here, should figure out how to pass them also
         # TODO: make sysbench run with dynamic values
         # TODO: make sysbench different possible kind of runs
-        # TODO: check this implementation; it seems to be buggy
 
         # Created sysbench DB
         db_name = "sysbench_test_db"

--- a/test/test_run_benchmark.py
+++ b/test/test_run_benchmark.py
@@ -7,7 +7,8 @@ class TestRunBenchmark:
     """
 
     def test_get_sock(self, return_run_benchmark_obj):
-        assert "sock" in return_run_benchmark_obj.get_sock()
+        for basedir in return_run_benchmark_obj.basedir:
+            assert "sock" in return_run_benchmark_obj.get_sock(basedir=basedir)
 
     def test_get_mysql_conn(self, return_run_benchmark_obj):
         assert "mysql" in return_run_benchmark_obj.get_mysql_conn()

--- a/test/test_run_benchmark.py
+++ b/test/test_run_benchmark.py
@@ -19,4 +19,5 @@ class TestRunBenchmark:
             assert return_run_benchmark_obj.create_db(db_name="test_run_benchmark_db", basedir=basedir) is True
 
     def test_run_sysbench(self, return_run_benchmark_obj):
-        assert return_run_benchmark_obj.run_sysbench() is True
+        for basedir in return_run_benchmark_obj.basedir:
+            assert return_run_benchmark_obj.run_sysbench(basedir=basedir) is True

--- a/test/test_run_benchmark.py
+++ b/test/test_run_benchmark.py
@@ -11,10 +11,12 @@ class TestRunBenchmark:
             assert "sock" in return_run_benchmark_obj.get_sock(basedir=basedir)
 
     def test_get_mysql_conn(self, return_run_benchmark_obj):
-        assert "mysql" in return_run_benchmark_obj.get_mysql_conn()
+        for basedir in return_run_benchmark_obj.basedir:
+            assert "mysql" in return_run_benchmark_obj.get_mysql_conn(basedir=basedir)
 
     def test_create_db(self, return_run_benchmark_obj):
-        assert return_run_benchmark_obj.create_db("test_run_benchmark_db") is True
+        for basedir in return_run_benchmark_obj.basedir:
+            assert return_run_benchmark_obj.create_db(db_name="test_run_benchmark_db", basedir=basedir) is True
 
     def test_run_sysbench(self, return_run_benchmark_obj):
         assert return_run_benchmark_obj.run_sysbench() is True

